operate/qa/integration-tests/src/test/java/io/camunda/operate/metric/UsageMetricIT.java

```diff
@@ -35,6 +37,8 @@
 @SpringBootTest(
     classes = {
       TestApplication.class,
+      UnifiedConfiguration.class,
+      UnifiedConfigurationHelper.class,
```

Comment:
Same here. Can be moved to TestApplication.

Reply:
Done